wrangler.toml

```diff
@@ -37,28 +37,31 @@ route = {pattern = "validator-status.cheqd.net/*", zone_id = "88e06eefedbbd140a1
 ENVIRONMENT = "production"
 # GraphQL API endpoint for target network. Must be sourced from a BigDipper instance.
 GRAPHQL_API = "https://explorer-gql.cheqd.io/v1/graphql"
+WEBHOOK_URL = "https://hooks.zapier.com/hooks/catch/{abc}/{123}/"
```

Comment:
This will _literally_ pass the value you have above to Cloudflare while it's deploying. So while it won't throw an error per se deploying, when it calls the webhook the worker will fail. 

The correct way to do this is to set `WEBHOOK_URL = ""` and the provide the value via the Github Actions deploy.yaml pipeline like described here: https://github.com/cloudflare/wrangler-action#configuration